packages/@stylexjs/babel-plugin/src/utils/evaluate-path.js

```diff
@@ -175,14 +175,17 @@ function evaluateThemeRef(
 
     const { debug, enableDebugClassNames } = state.traversalState.options;
 
-    const varSafeKey = (
-      key[0] >= '0' && key[0] <= '9' ? `_${key}` : key
-    ).replace(/[^a-zA-Z0-9]/g, '_');
+    const varSafeKey =
+      key === '__themeName__'
+        ? ''
+        : (key[0] >= '0' && key[0] <= '9' ? `_${key}` : key).replace(
+            /[^a-zA-Z0-9]/g,
+            '_',
+          ) + '-';
```

Comment:
This is a fix for dev/debug mode. The way theming is currently designed to work is broken in dev because this generates bad class names that don't map to anything in the generated CSS